index.html

```diff
@@ -1 +1,152 @@
-hello world
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Epilogue:wght@200;400;500;600;700&family=IBM+Plex+Sans:ital,wght@0,100;0,200;0,300;0,400;0,700;1,200;1,400&display=swap" rel="stylesheet">
+    <link href="style.css" rel="stylesheet">
+    </link>
+    <title>Soyeon Kwon's Resume</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <div id="info">
+            <h1>Soyeon<br>Kwon</h1>
+            <h3>User Experience & Visual Designer</h3>
```

Comment:
Here, you've skipped from an `h1` to an `h3` -- and no heading levels should be skipped. See the [best practices examples](https://mica-web.github.io/learn/html/best-practices#heading-tags-are-used-semantically-not-simply-for-presentation).